pandas/core/strings.py

```diff
@@ -439,6 +446,14 @@ def str_extract(arr, pat, flags=0):
     2    NaN
     dtype: object
 
+    Specify ``expand=False`` to return Series.
+
+    >>> Series(['a1', 'b2', 'c3']).str.extract('[ab](\d)', expand=False)
+    0      1
+    1      2
+    2    NaN
+    dtype: object
+
```

Comment:
I think it'd be good to change this example to showcase `expand=False` when it actually has multiple groups, i.e., 

```
>>> Series(['a1', 'b2', 'c3']).str.extract('([ab])(\d)', expand=False)
0        [a, 1]
1        [b, 2]
2    [nan, nan]
Name: [0, 1], dtype: object
```

I'd also move this one example down, so we'd have:

"A pattern with more than one group will return a DataFrame."

"But you can specify ``expand=False`` to return Series."